### PR TITLE
Include basic instructions on how to use in README

### PR DESCRIPTION
**Pull request by duduribeiro**
This commit adds a basic instruction on how to use the cops in the
README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
 # RuboCop minitest
```
Comment (koic) [inline README.md]: Can you update to `RuboCop Minitest`? I had a typo in the name.

**Comment by koic** [comment]
I'd like to release gem after merging #6. At that time I will merge this PR. Please wait a moment and thanks for your activities.

**Comment by duduribeiro** [comment]
Perfect!.

Thanks so much for the help @koic

**Comment by koic** [comment]
Thanks!

**Pull request merged**